specification/support/resource-manager/Microsoft.Support/preview/2019-05-01-preview/support.json

```diff
@@ -0,0 +1,1502 @@
+                        "schema": {
+                            "$ref": "#/definitions/ProblemClassificationsListResult"
+                        }
+                    },
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "#/definitions/ExceptionResponse"
+                        }
+                    }
+                },
+                "x-ms-pageable": {
+                    "nextLinkName": null
+                },
+                "x-ms-examples": {
+                    "ListProblemClassificationsForService": {
+                        "$ref": "./examples/ListProblemClassifications.json"
+                    }
+                }
+            }
+        },
+        "/providers/Microsoft.Support/services/{serviceName}/problemClassifications/{problemClassificationName}": {
+            "get": {
+                "tags": [
+                    "ProblemClassifications"
+                ],
+                "description": "Gets the details of a specific problem classification for a specific Azure service.",
+                "operationId": "ProblemClassifications_Get",
+                "parameters": [
+                    {
+                        "name": "serviceName",
+                        "in": "path",
+                        "description": "Name of Azure service available for support.",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "problemClassificationName",
+                        "in": "path",
+                        "description": "Name of problem classification.",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Successfully retrieved problem classification details.",
+                        "schema": {
+                            "$ref": "#/definitions/ProblemClassification"
+                        }
+                    },
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "#/definitions/ExceptionResponse"
+                        }
+                    }
+                },
+                "x-ms-examples": {
+                    "GetIssueClassification": {
+                        "$ref": "./examples/GetProblemClassification.json"
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/providers/Microsoft.Support/checkNameAvailability": {
```

Comment:
Why is this API necessary? What's the scope of the name that you're checking for availability - is this something that must be globally unique?

Reply:
This is based on the feedback from Mark Simms/Gaurav/Kris. I will forward you that thread. The ticket is unique at the subscription scope.